src/views/onboarding/tutorial-view/server.ts

```diff
@@ -47,8 +47,11 @@ export async function getOnboardingTutorialProps(
 		(collection: ApiCollection) =>
 			collection.slug === `${onboardingData.slug}/${collectionFilename}`
 	)
-	const currentTutorialReference = currentCollection?.tutorials.find((t) =>
-		t.slug.endsWith(tutorialFilename)
```

Comment:
This was too loose of a match. We had a situation where two tutorials in a single collection had the same filename end. [(see line 13/14)](https://github.com/hashicorp/tutorials/blob/main/content/collections/onboarding/hcp-vault-week-3.yml#L13-L15)
<img width="734" alt="Screenshot 2023-07-10 at 11 01 46 AM" src="https://github.com/hashicorp/dev-portal/assets/36613477/a2c54ee5-47fc-45fd-8c8e-3ce23cccd536">